e2e/main.e2e.spec.ts

```diff
@@ -3,8 +3,12 @@ import path from 'path';
 import {
   ElectronApplication, BrowserContext, _electron, Page, Locator
 } from 'playwright';
-import { test, expect } from '@playwright/test';
+import { test } from '@playwright/test';
 import { createDefaultSettings, playwrightReportAssets } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import { K8sPage } from './pages/k8s-page';
+import { WslPage } from './pages/wsl-page';
```

Comment:
Yep, I thought on the same approach, but the set up take a few seconds and it will impact a bit on the test time.
Only main spec will be affected by this approach, so I'm assuming this is ok.